solidity/test/unit/Greeter.t.sol

```diff
@@ -29,6 +29,14 @@ contract GreeterForInternalCallsTest is InternalCallsWatcherExtension, Greeter {
   }
 }
 
+contract GreeterForTest is Greeter {
+  constructor(string memory _greeting, IERC20 _token) Greeter(_greeting, _token) {}
+
+  function updateLastGreetingSetTime(uint256 _timestamp) external virtual {
+    _updateLastGreetingSetTime(_timestamp);
+  }
```

Comment:
i wonder if this would work, without requiring having 2 contracts ForTest
```solidity
  function call_updateLastGreetingSetTime(uint256 _timestamp) external virtual {
    super._updateLastGreetingSetTime(_timestamp);
  }
  ```
  if this one ignores the upper override, i'd say its better